### DISCUSSION TITLE
Regarding accessibility of Mantine documentation

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#2098] [Feature requests]
## Suggest

Mantine's documentation delights users with its many examples and modern designs. Very nice.

But there is one downside.

It works differently than expected in keyboard accessibility.

Usually, if you click on a specific area and press the tab, you can access the tab to that area, but this is not the case with Mantine's document.

You may not understand what I'm saying, but I'll explain in detail.

## Example (Github)

![image](https://user-images.githubusercontent.com/27342882/184466432-0e245aa8-fc40-4327-882c-ad80f8f4362b.png)

Click on the area of ​​the photo on GitHub with the mouse and press the tab key to move the focus to the closest focusable area.

![image](https://user-images.githubusercontent.com/27342882/184466451-11c158ed-626b-4763-9696-040d291fc3a1.png)

## Example (Mantine Docs)

However, mantine's documentation doesn't work this way.

![image](https://user-images.githubusercontent.com/27342882/184466486-bd5e7ec7-1acf-4c47-98ba-212b96721b96.png)

Even if you click on another area, the logo area will have focus first.

## Etc

This may not be an important task, but it is suggested for a good user experience! thank!

**Comment by ryuhangyeong** [comment]
I know a very simple way to solve this problem. I'll post a PR if you like!

**Reply by rtivital** [reply]
Yeah, sure, feel free to send a PR
